Simplenote/Classes/SPCardViewController.swift

```diff
@@ -0,0 +1,61 @@
+import UIKit
+
+// MARK: - SPCardViewController
```

Comment:
Nipticky: Mind adding a brief description of what's the purpose of this VC?

Reply:
Good point! Will add a description 👍